homeassistant/const.py

```diff
@@ -680,7 +684,6 @@
 PRESSURE: Final = "pressure"
 VOLUME: Final = "volume"
 TEMPERATURE: Final = "temperature"
-SPEED: Final = "speed"
```

Comment:
This needed to be kept (my bad for the bad code suggestion:
SPEED: Final = "speed"